samples/system-test/quickstart.test.js

```diff
@@ -15,44 +15,51 @@
 
 'use strict';
 
-const proxyquire = require(`proxyquire`).noPreserveCache();
+const proxyquire = require(`proxyquire`);
 const sinon = require(`sinon`);
-const test = require(`ava`);
+const assert = require('assert');
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 
-test.before(tools.stubConsole);
-test.after.always(tools.restoreConsole);
+describe('QuickStart', () => {
+	before(() => {
```

Comment:
it's fine to drop the curly braces and keep this a one liner.  Same for the `afterEach` below.

Reply:
Change done